lang/java/avro/src/test/java/org/apache/avro/TestReadingWritingDataInEvolvedSchemas.java

```diff
@@ -0,0 +1,268 @@
+package org.apache.avro;
```

Comment:
@harshach my branch (actually, the same commit) was updated 10th of April, don't know what else to do. Waiting for your review and acceptance/merge into master.
The Apache License was added to my new file, and the reason the // is there in the file is in order to preserve formatting across IDEs (My Eclipse has been acting erratically lately - Neon upgrade issue)